src/Analysis/GWASApp/Utils/StateManagement/InitialState.js

```diff
@@ -11,6 +11,7 @@ const initialState = {
   finalPopulationSizes: [],
   selectionMode: '',
   messages: [],
+  selectedTeamProject: 'temporary-test-teamproject01', // TODO - remove and leave '' when teamproject selection is ready
```

Comment:
teamproject selection will be done, according to design, through a dropdown in step1 of the workflow.